frontend/src/pages/MapView/index.jsx

```diff
@@ -133,12 +73,15 @@ class MapView extends Component {
                 searchHandler={this.searchHandler}
                 changeHandler={this.changeHandler}
                 searchSuggestions={this.state.searchSuggestions}
+                showSearchSuggestions={this.state.showSearchSuggestions}
                 inputValue={this.state.inputValue}
               />
             </div>
           </div>
           <div className="cardContent">
-            {this.state.showResults && searchResults.map(this.renderCards)}
+            {this.state.showResults &&
+              this.state.searchResults.length > 0 &&
```

Comment:
nit: just `this.state.searchResults.length` instead of `this.state.searchResults.length > 0`